Add information for 2018, 2019, and 2020 Terraform Enterprise releases (#2138)
hashicorp/terraform-website: content/enterprise/release/index.mdx

# Terraform Enterprise Releases

~> The next release `v202202-1` is scheduled for February 25th, 2022.

> We release a new Terraform Enterprise version each month. The releases from the current calendar year, plus the last required release, are listed in the table below. You can find previous releases in the sidebar.

Reviewer: Along similar lines to using active voice:

```
The table below lists releases from the current calendar year and denotes which releases are required. 
```

# Terraform Enterprise Releases

~> The next release `v202202-1` is scheduled for February 25th, 2022.

> A new Terraform Enterprise version is released each month. The releases from the current calendar year, plus the last

Reviewer 2: ```suggestion
We release a new Terraform Enterprise version each month. The releases from the current calendar year, plus the last
```

> required release, are listed in the table below. Previous releases can be found on the sidebar.

Reviewer 2: I'm assuming we are only listing the ones that are still supported, yes? Push back if I'm wrong :p 

```suggestion
required release, are listed in the table below. You can find previous supported releases in the sidebar.
```
